app/ci_program.py

```diff
@@ -105,7 +105,7 @@ def commandLoop(self):
       while not len(cmdList):
         for i in range(5):
           ch = window.cursorWindow.getch()
-          if ch == curses.ascii.ESC:
+          if ch == app.curses_util.KEY_ESCAPE:
```

Comment:
Hmm yeah you're right. I was thinking that it would be okay since I assigned the same value to the new variable, but this would break if we end up changing the value later on.